jsk_fetch_robot/jsk_fetch_gazebo.noetic.repos

```diff
@@ -0,0 +1,9 @@
+repositories:
+  ZebraDevs/fetch_gazebo:
+    type: git
+    url: git@github.com:ZebraDevs/fetch_gazebo
+    version: gazebo11
+  GT-RAIL/robot_pose_publisher:
```

Comment:
which launch file uses this node? https://github.com/search?q=repo%3AZebraDevs%2Ffetch_gazebo%20robot_pose_publisher&type=code